### DISCUSSION TITLE
Way to turn properties into X | null?

### DISCUSSION THREAD

**Discussion by xaviergonz** [millsp/ts-toolbelt#182] [Wiki]
Is there something like `O.Nullable` but that actually makes keys turn into `V | null`?
e.g.
```
interface A { x: number; y: number }
type AN = O.NullNullable<A, "x">
// AN is { x: number | null; y: number }
```

**Comment by millsp** [comment]
Yes, you can use [Update](https://pirix-gh.github.io/ts-toolbelt/modules/_object_update_.html#update)

**Comment by xaviergonz** [comment]
Is this supposed to work then? It doesn't for me
```ts
interface A {y: number}
type C = O.Update<A, "y", A.x | null>
const c: C = {y: 4} // complains about 'y' being a number
```

**Comment by xaviergonz** [comment]
Nevermind, didn't work on 4.1.0 but works on 4.3.0
